web/app.fluidity.money/app/components/NotificationSubscription.tsx

```diff
@@ -145,9 +145,14 @@ export const NotificationSubscription = ({
 
   useEffect(() => {
     if (rawAddress) {
-      const { emitEvent } = DSSocketManager({
-        onCallback: handleClientListener,
-      });
+      const protocol = window.location.protocol === "http:" ? "ws:" : "wss:";
+      const port = protocol === "ws:" ? "3000" : "80";
```

Comment:
WSS would be :443 right? Because the scheme is `https://`.
Regardless this is certainly a bit cursed.
You should be able to omit the port if it's the default for the scheme.